Author: Diane Trout <diane@ghic.org>
Forwarded: not-needed
Description: Use Debian packaged marked and marked-man to build Documentation

--- a/Makefile
+++ b/Makefile
@@ -103,13 +103,15 @@
 
 marked: node_modules/.bin/marked
 
-node_modules/.bin/marked:
-	node bin/npm-cli.js install marked --no-global --no-timing --no-save
+#node_modules/.bin/marked:
+#	node bin/npm-cli.js install marked --no-timing --prefer-offline
+.PHONY: node_modules/.bin/marked
 
 marked-man: node_modules/.bin/marked-man
 
-node_modules/.bin/marked-man:
-	node bin/npm-cli.js install marked-man --no-global --no-timing --no-save
+#node_modules/.bin/marked-man:
+#	node bin/npm-cli.js install marked-man --no-timing --prefer-offline
+.PHONY: node_modules/.bin/marked-man
 
 test: docs
 	node bin/npm-cli.js test
